core/src/test/java/tech/tablesaw/mapping/BooleanMapUtilsTest.java

```diff
@@ -0,0 +1,38 @@
+package tech.tablesaw.mapping;
+
+import org.junit.Test;
+import tech.tablesaw.api.BooleanColumn;
+
+import static org.junit.Assert.assertEquals;
+
+public class BooleanMapUtilsTest {
+    @Test
+    public void testAdd() {
+        BooleanColumn booleanColumn = new BooleanColumn("");
```

Comment:
Most columns have a constructor like:

    public DoubleColumn(String name, double[] arr) {

I see `BooleanColumn` does not. It'd probably be good for you to add one so that creating these columns would be simpler like:

    BooleanColumn booleanColumn = new BooleanColumn("col1", new boolean[]{ true });